.github/workflows/release.yaml

```
@@ -79,46 +81,92 @@ jobs:
           fork-user: prql-bot
 
   build-deb-package:
-    # TODO: currently this doesn't publish it as an artifact; would be a welcome contribution
-    # to add that (and eventually as a release asset)
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - x86_64-unknown-linux-musl
+          - aarch64-unknown-linux-musl
+    needs: build-prqlc
+    permissions:
+      contents: write
     steps:
-      - name: 📂 Checkout code
-        uses: actions/checkout@v4
-      - name: 👷 Build prql-compiler
-        run: cargo build --release -p prqlc
+      - name: Download a Build Artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: prqlc-${{ matrix.target }}
       - name: Copy files into .deb package
         run: |
+          tar -xf prqlc-*.tar.gz
           mkdir -p .debpkg/usr/bin
-          cp target/release/prqlc .debpkg/usr/bin/prqlc
+          mv prqlc .debpkg/usr/bin/prqlc
           chmod +x .debpkg/usr/bin/prqlc
+      - name: Set variables
```

Let's either use a descriptive name or no name

```suggestion
      - name: Set arch variable
```